src/SFA.DAS.EmployerFinance/Http/HttpClientWrapper.cs

```diff
@@ -8,7 +8,7 @@
 using Newtonsoft.Json;
 using SFA.DAS.NLog.Logger;
 
-namespace SFA.DAS.Http
+namespace SFA.DAS.EmployerFinance.Http
```

Comment:
How come the `HttpClientWrapper` has been moved to finance instead of making use of the das-shared/SFA.Das.Http package? Think you mentioned this as something you didn't want to do but had to do for simplicity?

Reply:
There's no correlation between the HTTP client in SFA.Das.Http and our solution. I think in time the Finance project should be changed to work with the existing nuget package. But I don't think this is work for now.